### DISCUSSION TITLE
What updated?

### DISCUSSION THREAD

**Discussion by MasterInQuestion** [liriliri/eruda#446] [General]
[ https://github.com/liriliri/eruda/compare/v3.2.1...v3.2.2](https://github.com/liriliri/eruda/compare/v3.2.1...v3.2.2)
    “What updated..?
    Updated version number.”

**Comment by surunzi** [comment]
Fixed broken built-in plugins.

https://github.com/liriliri/eruda-orientation
https://github.com/liriliri/eruda-geolocation

**Reply by MasterInQuestion** [reply]
More carefully view revealed then...
    The repository's structure is not very straight forward, primarily for this limitation:
    https://github.com/orgs/community/discussions/39375#discussioncomment-8665392
